src/solution_part2.kt

```diff
@@ -1,3 +1,18 @@
 fun main(){
- print("I love Kotlin!!!!")
+ val player1= 80
+var healthPointP1= if (player1 in 50..90) {
+  "Player1 you have $player1 health points so your health point is fine!!"
+}else{
+ " Player1 you have $player1 health points so your health point is awful!!"
+}
+ println(healthPointP1)
+
+    val player2= 30
+    var healthPointP2= if (player2 in 50..90) {

```

Comment:
less than 90 so you shouldn't include 90 in the range 
var healthPointP2= if (player2 in 50..89) {